src/Sql/dbo/Stored Procedures/Organization_Create.sql

```diff
@@ -44,7 +44,12 @@
     @UseScim BIT = 0,
     @UseCustomPermissions BIT = 0,
     @UseSecretsManager BIT = 0,
-    @Status TINYINT = 0
+    @Status TINYINT = 0,
+    @UsePasswordManager BIT = 1,
+    @SmSeats INT,
```

Comment:
Since the columns are nullable then we should set a default value here for all new `INT` columns to `NULL` in case no parameter is received